src/components/GenericWarningModal/GenericWarningModal.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Text,
+  HStack,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from "@chakra-ui/react"
+import { ModalCloseButton } from "@opengovsg/design-system-react"
+import parse from "html-react-parser"
+
+interface ModalProps {
+  isOpen: boolean
+  onClose: () => void
+  displayTitle: string
+  displayText: string
+  children: JSX.Element | JSX.Element[]
+}
+
+export const GenericWarningModal = ({
```

Comment:
nit: could we change the name to be `WarningModal`? somehow having the generic sounds weird AHHAHA

Reply:
Sure, changed in https://github.com/isomerpages/isomercms-frontend/pull/941/commits/89a80125e5a6b606510663d0f3f3003c382fb0e1